### PR TITLE
Some housekeeping: .gitignore, test targets, ...

### PR DESCRIPTION
**Pull request by egrimley-arm**

### CODE REVIEW

**Review by mathias-arm — APPROVED** [review]

File: workspaces/linux-runtime/Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ cargo-features = ["resolver"]
 [workspace]
 exclude = [
   "crates/icecap/sysroot/libc",
+  "crates/third-party/rust-psa-crypto",
   "crates/third-party/wasmi",
   "crates/third-party/rust-psa-crypto",
```
Comment (mathias-arm) [inline workspaces/linux-runtime/Cargo.toml:20]: Duplicate entry

**Comment by mathias-arm** [comment]
Your change in seL4 was pushed on the `icecap` branch while Veracruz is currently tracking `realmos` (see [.gitmodules](https://github.com/veracruz-project/veracruz/blob/main/.gitmodules#L48)). I have cherry-picked your change into [realmos](https://gitlab.com/arm-research/security/icecap/seL4/-/commits/realmos) in [0a640d04](https://gitlab.com/arm-research/security/icecap/seL4/-/commit/0a640d04220178fa4d4e2f521d54aea3989ecf82).

**Comment by egrimley-arm** [comment]
> Your change in seL4 was pushed on the `icecap` branch while Veracruz is currently tracking `realmos` (see [.gitmodules](https://github.com/veracruz-project/veracruz/blob/main/.gitmodules#L48)). I have cherry-picked your change into [realmos](https://gitlab.com/arm-research/security/icecap/seL4/-/commits/realmos) in [0a640d04](https://gitlab.com/arm-research/security/icecap/seL4/-/commit/0a640d04220178fa4d4e2f521d54aea3989ecf82).

I'll keep this PR as pure "housekeeping" and assume that change will get picked up by some other commit to `veracruz` then.

**Review by dominic-mulligan-arm — APPROVED** [review]

**Comment by dominic-mulligan-arm** [comment]
+1 reached, merging.

**Pull request merged**